apps/builder/app/builder/features/settings-panel/props-section/animation/animation-section.tsx

```diff
@@ -103,12 +103,29 @@ const InsetValueInput = ({
 }: {
   id: string;
   value: InsetUnitValue;
-  onChange: (value: InsetUnitValue) => void;
+  onChange: ((value: undefined, isEphemeral: true) => void) &
+    ((value: InsetUnitValue, isEphemeral: boolean) => void);
 }) => {
   const [intermediateValue, setIntermediateValue] = useState<
     StyleValue | IntermediateStyleValue
   >();
 
+  const handleEphemeralChange = (styleValue: unknown | undefined) => {
```

Comment:
why unknown? Don't we have a guaranteed type from css value input?

Reply:
To make implmenetation of underlying controls a little bit easier